### DISCUSSION TITLE
List of available scopes/sights for weapons

### DISCUSSION THREAD

**Discussion by SoveriegnTurkey** [janstehno/cotw-companion#25] [Ideas]
A simple addition to the weapon infortion section, would be what sihgts are available. While many have acces to the same scopes, there are some that use specific scopes such as the helios for the .30-.30 and 
.45-.70 rifles. For rifles that have specific tritum sights, they can be simply listed as "tritium sights" since any weapon that has them has a specific version.

**Comment by janstehno** [comment]
I see. The initial tought of yours (many weapons have same scopes) is the reason I did not added it. But I can look into it, shouldn't be hard to implement.

**Comment by janstehno** [comment]
So I have thought about this and I still think it is not needed. The in-game feature called "available equipment" is enough to compensate for not having this information in the app in my opinion. This will not be implemented for now.
